v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: false
+---
+
+A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication, which requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, or use [OpenSSL commands](https://wiki.openssl.org/index.php/).
+
+<div class="filters filters-big clearfix">
```

Comment:
Can we try putting these toggles at the top of the page, like we do [here](https://www.cockroachlabs.com/docs/stable/build-a-go-app-with-cockroachdb.html)?

Also, I'd suggest trying this text:
- Use Cockroach Commands
- Use OpenSSL Commands

We may have to force wider toggles. See https://github.com/cockroachdb/docs/blame/master/v1.1/build-a-python-app-with-cockroachdb.md#L10 for an example.

Reply:
@jseldess The reason I put the toggles after the first paragraph is that the paragraph is common to both. So if we move the toggles to the top, I will have to repeat the para on both pages. Not that I mind doing that. Thoughts?

Reply:
Reworded toggle text, increased width.